### PR TITLE
chore: Add more typescript fixes

### PR DESCRIPTION
**Pull request by tommasini**
## **Description**

Before: 
Found 225 errors in 77 files.

After:
Found 157 errors in 50 files.

## **Related issues**

Fixes:

## **Manual testing steps**

1. Go to this page...
2.
3.

## **Pre-merge author checklist**

- [ ] I’ve followed [MetaMask Contributor Docs](https://github.com/MetaMask/contributor-docs) and [MetaMask Mobile Coding Standards](https://github.com/MetaMask/metamask-mobile/blob/main/.github/guidelines/CODING_GUIDELINES.md).
- [ ] I've completed the PR template to the best of my ability
- [ ] I’ve included tests if applicable
- [ ] I’ve documented my code using [JSDoc](https://jsdoc.app/) format if applicable
- [ ] I’ve applied the right labels on the PR (see [labeling guidelines](https://github.com/MetaMask/metamask-mobile/blob/main/.github/guidelines/LABELING_GUIDELINES.md)). Not required for external contributors.

## **Pre-merge reviewer checklist**

- [ ] I've manually tested the PR (e.g. pull and build branch, run the app, test code being changed).
- [ ] I confirm that this PR addresses all acceptance criteria described in the ticket it closes and includes the necessary testing evidence such as recordings and or screenshots.

### CODE REVIEW

**Review by brianacnguyen — DISMISSED** [review]

File: app/components/UI/AssetOverview/AboutAsset/AboutAsset.tsx [hunk 1]
```diff
@@ -22,7 +22,6 @@ const AboutAsset = ({ asset, chainId }: AboutAssetProps) => {
   const { styles } = useStyles(styleSheet, {});
   const locale: keyof TokenDescriptions = i18n.locale;
   const skeletonProps = {
-    width: '100%',
```
Comment (kylanhurt) [inline app/components/UI/AssetOverview/AboutAsset/AboutAsset.tsx:25]: @tommasini Why did we remove this?

File: app/components/UI/WalletAccount/WalletAccount.test.tsx [hunk 2]
```diff
@@ -2,9 +2,12 @@
 import React from 'react';
 import { fireEvent, waitFor } from '@testing-library/react-native';
 import { KeyringTypes } from '@metamask/keyring-controller';
+import { Hex } from '@metamask/utils';
```
Comment (kylanhurt) [inline app/components/UI/WalletAccount/WalletAccount.test.tsx:5]: @tommasini Is this type even used?

Comment (tommasini) [reply app/components/UI/WalletAccount/WalletAccount.test.tsx:5]: Yes! [Here](https://github.com/MetaMask/metamask-mobile/blob/5d05f0a5b2463e51f34e5753805036f5d602621d/app/components/UI/WalletAccount/WalletAccount.test.tsx#L26) and [here](https://github.com/MetaMask/metamask-mobile/blob/5d05f0a5b2463e51f34e5753805036f5d602621d/app/components/UI/WalletAccount/WalletAccount.test.tsx#L32)

File: app/components/UI/AssetOverview/AboutAsset/AboutAsset.tsx (hunk 1 above)
Comment (tommasini) [reply app/components/UI/AssetOverview/AboutAsset/AboutAsset.tsx]: Added back nice catch! This component is actually not used but will deep dive to try to fix the typescript issue better!

**Review by smilingkylan — DISMISSED** [review]
LGTM

**Review by smilingkylan — APPROVED** [review]

**Pull request merged**